tools/accounts_cli.go

```diff
@@ -0,0 +1,259 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/codegangsta/cli"
+)
+
+const (
+	prd_host   = "https://api.tidepool.org"
+	stg_host   = "https://stg-api.tidepool.org"
+	dev_host   = "https://dev-api.tidepool.org"
+	local_host = "http://localhost:8009"
+
+	tp_server_name   = "x-tidepool-server-name"
+	tp_server_secret = "x-tidepool-server-secret"
+	tp_token         = "x-tidepool-session-token"
+)
+
+type admin struct {
+	client *http.Client
+	token  string
+	env    string
+}
+
+func main() {
+
+	app := cli.NewApp()
+
+	app.Name = "User Role"
+	app.Usage = "Internal tool to add update and find users by role type"
+	app.Version = "0.0.1"
+	app.Author = "Jamie"
+	app.Email = "jamie@tidepool.org"
+
+	app.Commands = []cli.Command{
+
+		//e.g. update -e testie@user.org -r clinic
+		{
+			Name:      "update",
+			ShortName: "u",
+			Usage:     `update and existing account to have a role`,
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "userid",
+					Usage: "userid of the account your updating",
+				},
+				cli.StringFlag{
+					Name:  "role",
+					Value: "clinic",
+					Usage: "the role that is being given to the user, defaults to `clinic`",
+				},
+				cli.StringFlag{
+					Name:  "env",
+					Usage: "the environment we are running against",
+				},
+			},
+			Action: updateAccount,
+		},
+		//e.g. find -r clinic
+		{
+			Name:      "find-account",
```

Comment:
Can we combine this into the update-account? So, you update account with an email, rather than a user id?